main.go

```diff
@@ -298,6 +303,28 @@ func getInstances(ctx context.Context) (*config.APIConfig, *bitbucket.Handler, *
 		)
 	}
 
+	var cloudsourceClient cloudsourceapi.Client
+	{
+		tokenSource, err := google.DefaultTokenSource(ctx, sourcerepo.CloudPlatformScope)
+		if err != nil {
+			log.Fatal().Err(err).Msg("Creating token source for cloud source repo client has failed")
+		}
+
+		sourcerepoService, err := sourcerepo.New(oauth2.NewClient(ctx, tokenSource))
+		if err != nil {
+			log.Fatal().Err(err).Msg("Creating source repo service has failed")
+		}
```

Comment:
Can't we initialize this inside `cloudsourceapi.NewClient` to keep things cleaner over here? `NewClient` can return `(Client, error)` so we can still handle errors properly.

Reply:
Good one. Initially I did it inside NewClient. I changed because I saw bigquery client doing it like this.
But I'll move it inside.

Reply:
Not sure why I did it differently for the bigquery client, we can encapsulate it there in a separate PR as well.